server/service/individualServices/RegardApplication.js

```diff
@@ -92,62 +94,68 @@ exports.regardApplication = function (applicationName, releaseNumber, user, xCor
                                     "successfully-connected": false,
                                     "reason-of-failure": "OL_UNKNOWN"
-                                });
-                            }else {
+                                }
+
+                            } else {
                                 if (result['status'] != 200) {
-                                    resolve({
+                                    statusForCreateLinkForInquiringServiceRecords = {
                                         "successfully-connected": false,
                                         "reason-of-failure": "OL_UNKNOWN"
-                                    });
-                                    break;
+                                    }
+
                                 } else if (result['status'] == 200 && !result['data']['client-successfully-added']) {
-                                    resolve({
+
+                                    statusForCreateLinkForInquiringServiceRecords = {
                                         "successfully-connected": false,
                                         "reason-of-failure": `OL_${result['data']['reason-of-failure']}`
-                                    });
-                                    break;
+                                    }
                                 } else {
                                     let operationServerUuidOfRecordOamRequest = "ol-2-1-0-op-s-is-004";
                                     let isOperationServerKeyUpdated = await operationKeyUpdateNotificationService.waitUntilOperationKeyIsUpdated(
                                         operationServerUuidOfRecordOamRequest,
                                         timestampOfCurrentRequest,
                                         waitTime);
                                     if (!isOperationServerKeyUpdated) {
-                                        resolve({
+                                        statusForCreateLinkForInquiringServiceRecords = {
                                             "successfully-connected": false,
                                             "reason-of-failure": "OL_MAXIMUM_WAIT_TIME_TO_RECEIVE_OPERATION_KEY_EXCEEDED"
-                                        });
-                                        break;
+                                        }
+
                                     } else {
-                                        resolve({
-                                            "successfully-connected": true
-                                        });
-                                        break;
+                                        statusForCreateLinkForInquiringServiceRecords = {
+                                            'successfully-connected': true
+                                        }
+
                                     }
                                 }
+                                break;
```

Comment:
Earlier the break statement was inside the previous else loop. what's the rational behind moving it out of the else scope ?
I hope the previous reason is "for loop that performs multiple attempt shall be stopped if the operation is success".